doc(aws): add details in metering setup (#338)
meshcloud/meshcloud-docs: docs/meshstack.aws.metering.md

`Anniversary` are taken into the calculation. In other words, line items with bill type `Purchase` and `Refund` are excluded.
* For each line item, we take the [effective cost](https://docs.aws.amazon.com/cur/latest/userguide/reservation-columns.html#r-E)
when available and [unblended cost](https://docs.aws.amazon.com/cur/latest/userguide/Lineitem-columns.html#l-U) otherwise

## IAM User Configuration

### Policies

In order for meshStack to process AWS Cost and Usage Reports, following policies are required:

1. **Bucket Access Policy**: This policy allows the Metering IAM user to fetch AWS cost and usage reports
2. **Organization Access Policy**: This policy allows the Metering IAM user to list all accounts in the organization
3. **Assume Role Policy**: This policy allows Metering IAM user to assume the IAM Role `MeteringRole`

<!--DOCUSAURUS_CODE_TABS-->
<!--Bucket Access Policy-->
```json
{
    "Version": "2012-10-17",
    "Statement": [
        {

>             "Effect": "Allow",

Reviewer: q: I suppose this was taken from the metering role in our dev env? We probably don't need all these permissions. Is it a predefined policy on AWS?

Author: Yes it was taken from the dev env, but the permissions are the same ones we use in our [terraform platform module](https://github.com/meshcloud/meshfed-release/blob/develop/deployment/deployments/tf-platform-modules/aws/meshcloud-cost-usage-reports-iam-user/iam.tf#L19-L53) in which we create a custom AWS policy

Author: As discussed in our call, the unnecessary permissions are removed, thanks!

                  access-key: "<AWS_ACCESS_KEY>"
                  secret-key: "<AWS_SECRET_KEY>"
          -}
          { access-key : Secret, secret-key : Secret }
      , cost-and-usage-report :
          {-
              Parameters required to process AWS cost and usage reports
                  name: The name of the report
                  bucket-name: The name of the S3 bucket where the report is written
                  bucket-region: The region of the S3 bucket
                  report-key-prefix: The report path prefix. Generated reports will be located at <bucket-name>/<report-key-prefix>
          -}
          { name : Text
          , bucket-name : Text
          , bucket-region : Text
          , report-key-prefix : Text
          }
      , organization-root-account-role :
          {- The role assigned to the metering IAM user -}
          Text
      , organization-root-account-external-id :
          {- The external ID passed to "sts:AssumeRole" when used in privileged context -}
          Optional Text
      , cost-item-write-batch-size :
          {- Cost items written per batch -}
          Natural
      , max-report-file-processing-retries :

>           {- The maximum number of attempts to process the report file -}

Reviewer: ```suggestion
          {- The maximum number of attempts to process a single file in a report -}
```

          Natural
      , report-file-process-retry-delay-millis :
          {- Time in milliseconds before an operation (report file processing) is retried -}
          Natural
      , max-bulk-insert-retries :

>           {- The maximum number of attempts to process a cost item batch -}

Reviewer: ```suggestion
          {- The maximum number of attempts to process a batch of cost items to the database -}
```

          Natural
      , bulk-insert-retry-delay-millis :
          {- Time in milliseconds before an operation (cost item batch processing) is retried -}
          Natural
      , cost-item-read-page-size :

>           {- Cost items retrieved per query -}

Reviewer: ```suggestion
          {- Cost items to read per query when generating tenant usage reports. This setting is useful to prevent read timeout errors in the case of accounts with a large number of cost items. -}
```
